java/generate_bazel_deps.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Generate Java dependencies for bazel.
+
+ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
+
+jar_file="/tmp/bazel_deps.jar"
+if [[ -e $jar_file ]]; then
+    echo "Use existing bazel_deps file from $jar_file."
+else
+    echo "Downloading bazel_deps file to $jar_file."
+    curl -L -o $jar_file "https://github.com/oferb/startupos-binaries/releases/download/0.1.01/bazel_deps.jar"
```

Comment:
What exactly are these dependencies for? The bazel dependencies seem to be attached to the 0.1.01 release but not the 0.1.02 release.

Reply:
These are the third-party libs (jar files) needed by Java. 
Previously in maven, we can define a lib simply with a (group id + artifact id + version) tuple. However, in Bazel, we also need the sha256 hash, the url, etc, which isn't convenient to manually compile.  See https://github.com/ray-project/ray/blob/fb88f7efe6a5158c77c31ec0df3ac603b6794b3f/java/third_party/workspace.bzl#L69
Thus we use this tool to auto-generate the dependencies.

Reply:
The version `0.1.01` is for the `bazel_deps.jar` tool. And `0.1.01` is the most recent version https://github.com/oferb/startupos-binaries/releases